IsraelHiking.Web/src/scss/_variables.scss

```diff
@@ -1,4 +1,5 @@
 $app-sidebar-width: 400px;
 $app-left-z-index: 1030;
 
-$app-button-size: 36px;
\ No newline at end of file
+$app-button-width: 36px;
+$app-button-height: 36px;
```

Comment:
Shouldn't this be 44px?

Reply:
It was 36px before I started, and I only changed the variable names. It’s not used in the buttons I’m working on, as I’ve set the button size to 44px in the common CSS file.